cmd/security-bootstrapper/entrypoint-scripts/ready_to_run_wait_install.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/dumb-init /bin/sh
+#  ----------------------------------------------------------------------------------
+#  Copyright (c) 2021 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+#  ----------------------------------------------------------------------------------
+
+# This is customized entrypoint script for other Edgex services.
+# In particular, it waits for the ReadyToRunPort raised to be ready to roll
+
+set -e
+
+# env settings
+. /edgex-init/.env-others
+
+echo "Script for waiting on security bootstrapping ready-to-run"
+
+# gating on the ready-to-run port
+echo "$(date) Executing dockerize with $@ waiting on tcp://${BOOTSTRAPPER_HOST}:$WAIT_TCP_PORT"
+/edgex-init/dockerize -wait tcp://"${BOOTSTRAPPER_HOST}":"$WAIT_TCP_PORT" -timeout "${WAIT_TIMEOUT_DURATION}"
+
+echo "$(date) Starting $@ ..."
+# If the user is trying to run the service with some arguments, then execute them directly
+# this can be useful if the user wants to run the service with non-default parameters
+if [ "${1:0:1}" = '-' ]; then
+    exec "$@"
+else
+#this is specifically for general edgex services such as core-data, app services, etc.
+    exec "$@" -cp=consul.http://"${CONSUL_HOST}":"$CONSUL_PORT" --registry --confdir=/res
```

Comment:
I have an issue around maintenance with this duplication of service command line option. Less discuss if we can have a better way to do this that isn't hidden in this script.

Reply:
ok, let's discuss this tomorrow.

Reply:
Let switch to overriding command also in compose file when we override entrypoint. That way this script can assume $@ is set appropriately and can simply `exec "$@"`  rather than the if/else

Reply:
ok